### ISSUE TITLE
Campo de sexo e aniversario separados?

### ISSUE THREAD

**Issue by franskaf** [claudiosanches/woocommerce-extra-checkout-fields-for-brazil#155]
Estive tentando inserir o campo de sexo e aniversario separados e não estou conseguindo.
Alguma sugestão?

**Comment by luizbills** [comment]
Como assim separados?

**Comment by franskaf** [comment]
Eu não consigo selecionar somente um ou outro para o check-out! Eu queria
apenas o campo de aniversário no cadastro!

Em dom, 29 de dez de 2019 13:24, luizbills <notifications@github.com>
escreveu:

**Comment by wilmerlm11** [comment]
Exato. As configurações destes campos deveriam ser separadas. Assim, possibilitaria a exibição só da data de nascimento, só do campo sexo ou os 2 juntos. É possível flexibilizar isso no plugin? Sei que tem como ativar a opção atual e remover um dos campos programaticamente, mas seria legal já vir configurável assim no plugin.
@luizbills @claudiosanches faz sentido?
Vlw gnt!
